spec/system/case_contacts/edit_spec.rb

```diff
@@ -23,6 +23,14 @@
       expect(case_contact.medium_type).to eq "letter"
       expect(case_contact.contact_made).to eq true
     end
+
+    it "fails across organizations" do
+      admin = create(:casa_admin, casa_org: create(:casa_org))
```

Comment:
I'm pretty sure the created admin and created casa org stick around after the test is run. If you wrap them in `let` they'll be deleted for sure.  

e.g

    let(:different_casa_org){create(:casa_org)}
    let(:admin){create(:casa_admin, casa_org: different_casa_org)}
    
    it "fails across organizations" do